sources/assets/Stride.Core.Assets.CompilerApp/build/Stride.Core.Assets.CompilerApp.targets

```diff
@@ -118,8 +118,8 @@
   <Target Name="_StridePrepareAssetCompiler">
     <PropertyGroup>
       <!-- First try NuGet layout, then git checkout -->
-      <StrideCompileAssetCommand Condition="'$(StrideCompileAssetCommand)' == '' And Exists('$(MSBuildThisFileDirectory)..\lib\net6.0-windows7.0\')">$(MSBuildThisFileDirectory)..\tools\net6.0-windows7.0\Stride.Core.Assets.CompilerApp.exe</StrideCompileAssetCommand>
-      <StrideCompileAssetCommand Condition="'$(StrideCompileAssetCommand)' == '' And Exists('$(MSBuildThisFileDirectory)..\Stride.Core.Assets.CompilerApp.csproj')">$(MSBuildThisFileDirectory)..\bin\$(Configuration)\net6.0-windows7.0\Stride.Core.Assets.CompilerApp.exe</StrideCompileAssetCommand>
+      <StrideCompileAssetCommand Condition="'$(StrideCompileAssetCommand)' == '' And Exists('$(MSBuildThisFileDirectory)..\lib\net6.0\')">$(MSBuildThisFileDirectory)..\tools\net6.0\Stride.Core.Assets.CompilerApp.exe</StrideCompileAssetCommand>
```

Comment:
That path doesn't look correct by the way. I can't find that `tools` folder in either the nuget package or the installed stride.

Reply:
i can't find the .exe too

Reply:
When building with net6.0-windows7.0 I get the tools folder ith the exe in it.
![image](https://github.com/stride3d/stride/assets/73259914/b4eae14f-f746-4d81-acd5-0e3feb83bddc)
 But not with just net6.0 and that was part of the problem it seemed.

Reply:
can you gen exe with the .net8.0? @Doprez

Reply:
it's the same issue, with net8.0-windows it works but with just net8.0 it doesn't.

Reply:
https://learn.microsoft.com/en-us/dotnet/core/deploying/
'exe' is a Windows thing so only appears when the platform is specified, otherwise you only get the 'cross-platform dll'

Reply:
That makes more sense. I guess the next question is why is it an exe instead of a loadable dll/nuget like other libraries to remove this dependancy. Converting the CompilerApp cross platform would probably be a fairly large task I am guessing.

Reply:
The way this is worked around for AssemblyProcessor is that it is loaded as an MSBuild task, but here it wouldn't be possible (VS would require it to target netstandard2.0).
I suggest reverting asset compiler to target windows for the moment to unblock compilation, and file an issue to discuss how to make it run cross platform correctly (linked with the Linux issue made by Caspian).

Reply:
Would it be possible instead to invoke `dotnet /path/to/dll`?